wom/src/main/scala/wom/expression/WomExpression.scala

```diff
@@ -23,23 +23,35 @@ trait WomExpression {
     * file paths, and should have all the essentials for determining if two `WomExpression` are conceptually the same.
     */
   def cacheString = sourceString
+
   def inputs: Set[String]
+
   def evaluateValue(inputValues: Map[String, WomValue], ioFunctionSet: IoFunctionSet): ErrorOr[WomValue]
+
   def evaluateType(inputTypes: Map[String, WomType]): ErrorOr[WomType]
+
   def evaluateFiles(inputValues: Map[String, WomValue], ioFunctionSet: IoFunctionSet, coerceTo: WomType): ErrorOr[Set[FileEvaluation]]
 
   /** Returns `true` if all file types within the specified `WomType` are optional. If not all the file types are
```

Comment:
Is this doc still correct? I guess I don't understand why a primitive type would be passed to this function based on these docs.

Reply:
Yes this is still correct, it's looking at the type of the output variable which could be a primitive.